components/brave_shields/resources/panel/components/main-panel/index.tsx

```diff
@@ -13,7 +13,7 @@ function MainPanel () {
   const { siteBlockInfo } = React.useContext(DataContext)
 
   const braveShieldsStatusText = splitStringForTag(siteBlockInfo?.isShieldsEnabled ? getLocale('braveShieldsUp') : getLocale('braveShieldsDown'))
-  const braveShieldsBlockedNote = splitStringForTag(getLocale('braveShieldsBlockedNote'))
+  const braveShieldsNote = splitStringForTag(siteBlockInfo?.isShieldsEnabled ? getLocale('braveShieldsBlockedNote') : getLocale('braveShieldsNOTBlockedNote'))
```

Comment:
nit: split the ternary on to multiple lines since it's a long one. Lint should catch this IMO so perhaps rules needs adjustment.